include/vrv/clef.h

```diff
@@ -57,6 +57,8 @@ class Clef : public LayerElement,
      */
     static int ClefId(data_CLEFSHAPE shape, char line, data_OCTAVE_DIS octaveDis, data_STAFFREL_basic place);
 
+    int PitchDistanceTo(Clef *clef);
```

Comment:
This might be totally my bad from last summer, but it seems like this function (and some of the other clef helper functions) might be mostly unused? If they're only called once or twice it might not be work having them

Reply:
Good point. It seems to only appear in `PitchInterface::AdjustPitchForNewClef`. I'm hesitant to remove it though just because it does seem like something that could be useful later.